js/data/question.jsonc

```diff
@@ -0,0 +1,599 @@
+    "aaci_multiplier": "変動ボーナス",
+    "rocket_barrage": "噴進弾幕発動率",
+    "plane_proficiency": "艦載機熟練度",
+    "prof_acc_bonus": "命中ボーナス",
+    "prof_crit_rate_bonus": "クリティカル率ボーナス",
+    "prof_crit_multiplier": "クリティカル率",
+    "prof_air_power": "制空値加算",
+    "night_contact": "夜間触接率",
+    "sas_improvement": "改修効果",
+	"impr_shelling_power": "砲戦火力",
+	"impr_shelling_acc": "砲戦命中",
+    "impr_shelling_evade": "砲戦回避",
+    "impr_night_power": "夜戦火力",
+    "impr_night_acc": "夜戦命中",
+    "impr_torpedo_power": "雷装",
+    "impr_torpedo_acc": "雷撃命中",
+    "impr_torpedo_evade": "雷撃回避",
+    "impr_asw_power": "対潜",
+    "impr_asw_acc": "対潜命中",
+    "impr_armour": "装甲",
+    "vs_installation_power": "対地火力",
+    "soft_skin": "ソフトスキン",
+    "pillbox": "砲台/トーチカ",
+	"iso_hime": "Iso Hime", // Who?
+    "harbour_summer": "港湾夏姫",
+    "supply_post_mod": "集積地棲姫補正",
+    "anchorage_post_mod": "泊地棲姫補正",
+    "dock_post_mod": "船渠棲姫補正",
+	
+    "sortie_air_power": "制空値(出撃)",
+    "defence_air_power": "制空値(防空)",
+    "high_altitude": "高高度迎撃",
+    "lbas_range": "半径",
+	"lbas_sortie_cost": "出撃コスト",
+    "lbas_fuel": "燃料",
+    "lbas_ammo": "弾薬",
+	
+	"import_export_data_format": "{link_db} と {link_aircalc} から出力されるデータをインポート。",
+	"deckbuilder": "デッキビルダー(本家)",
+	"aerial_combat_simulator": "制空シミュレータ",
+    "note_db_from_text": "メモ: 艦隊編成欄の「デッキビルダーから」にデッキビルダーデータを貼り付けることで、特定の艦隊だけインポートすることもできます。",
+	"lbas": "基地航空隊",
+	"import_support_normal": "道中支援",
+	"import_support_boss": "決戦支援",
+	"export_text": "艦隊&基地航空隊データを出力",
+	"open_in_deckbuilder": "デッキビルダーで開く",
+	"open_in_aircalc": "制空シミュで開く",
+	
+	"world": "海域",
+	"map": "エリア",
+	"edges": "マス",
+    "edges_desc": "例: 1,2,3",
+	"gauge": "攻略段階",
+	"map_hp": "ゲージ残量",
+	"difficulty": "難易度",
+	"diff_all": "全て",
+	"diff_A": "甲",
+	"diff_B": "乙",
+	"diff_C": "丙",
+	"diff_D": "丁",
+	"hq_level": "司令部レベル",
+	"ff_skip_comps": "Skip comps with same ship in Main Fleet?", // ?
+	"date": "検索範囲",
+	"submit": "読込",
+	"no_comps_found": "編成が見つかりませんでした",
+	"kcnav_error": "KCNavでエラーが発生しました",
+	"kcnav_timed_out": "KCNavでエラーが発生しました(タイムアウト)",
+	
+	"import_bonuses_dewy": "{link_dewy}から史実ボーナスをインポート:",
+	"dewy": "Dewy", // ok
+	"node": "マス",
+	"debuffed": "装甲破砕済",
+	"from_friend_fleet": "From Friend Fleet", // ?
+	"from_targeting_boss": "From Targeting Boss", // ?
+	"import_to_main_fleet": "Import to Main Fleet", // ?
+	"import_to_friend_fleets": "Import to Friend Fleets", // ?
+	"load_from_dewy": "Dewyから読み込み",
+	"load_from_kcnav": "KCNavから読み込み",
+	"no_data_found": "データが見つかりませんでした",
+    "no_ship_matches_found": "該当する船舶がありません",
+	"request_error_not_found": "Request error/Not found", // ok
+	"request_timed_out": "Request timed out", // ok
+	"ship": "艦",
+    "count": "Count", // ?
+    "max_low": "最大",
+    "min_high": "最小",
+    "import_value": "インポート",
+    "low_all": "最低値を採用",
+    "high_all": "最高値を採用",
+    "average_all": "平均値を採用",
+	"confirm": "決定",
+	
+	"warn_replay_import": "Warning: Replay import will overwrite existing data.", // ?

```

Comment:
https://kc3kai.github.io/kancolle-replay/battleplayer.html From here, load a Sample Code and click IMPORT INTO SIMULATOR. It is a warning that data imported this way will replace whatever you had before.